service/controller/resource/clusterconfigmap/desired.go

```diff
@@ -108,6 +108,9 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) ([]*cor
 
 			case "OpenStackCluster":
 				clusterValues["provider"] = "openstack"
+				clusterValues["dns"] = map[string]interface{}{
+					"service": "kube-dns",
```

Comment:
I don't quite follow why this is set here. Why not use the new default in `default-apps-openstack`?

Reply:
I don't mind, but did you see how it's all templated there? https://github.com/giantswarm/default-apps-openstack/blob/d04b8a5b8628076e3c6b79b54e3318d66e11a118/helm/default-apps-openstack/templates/net-exporter.yaml#L13